tests/api-mongo.test.js

```diff
@@ -412,3 +416,26 @@ describe('MongoDB Words', function () {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
+    it('should return response status of 404', (done) => {
+      const route = '/undefinedRoute';
+      getApiUrlRoute(route).end((_, res) => {
+        expect(res.status).to.equal(404);
+      });
+      done();
+    });
+  });
+
+  describe('/', () => {
+    it('should return hello world!', (done) => {
+      getApiUrlRoute().end(async (_, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.be.an('object');
+        expect(res.text).to.equal('Hello World!');
+      });
+      done();
```

Comment:
Same thing here, you want `done` to be placed inside your `end` callback.

Reply:
Placed it there but d github test failed it with some timeout stuff

Reply:
That's probably a good thing. You should investigate why making a simple `/` GET request to `http://igboapi.com/` is failing